zuul.d/jobs.yaml

```diff
@@ -25,3 +25,21 @@
       - ^docs_dev/.*$
       - ^docs_user/.*$
       - Gemfile
+
+- job:
+    name: data-plane-adoption-osp-17-to-extracted-crc-ironic
+    parent: data-plane-adoption-osp-17-to-extracted-crc
+    vars:
+      use_ceph: "false"
+      dpa_test_suite: "test-with-ironic"
+    irrelevant-files:
```

Comment:
I wonder whether we want to run it on all test changes or want to add more irrelevant files, e.g. tests roles of the services that Ironic doesn't depend on?

Reply:
E.g. if we just make an edit to Manila tests, there's probably no reason to run the Ironic job?

Reply:
+1 adding a third adoption job running everywhere will mean CI takes longer to run on PRs as well

Reply:
I agree that we should limit the relevant files.

Let's worry about getting the job to run at all first?